episodes/3-monitor-the-model.Rmd

```diff
@@ -189,12 +193,12 @@ Hint: A layer with `relu` activation, with `sigmoid` activation or no activation
 
 In our example we want to predict the sunshine hours in Basel (or any other place in the dataset) for tomorrow based on the weather data of all 18 locations today. `BASEL_sunshine` is a floating point value (i.e. `float64`). The network should hence output a single float value which is why the last layer of our network will only consist of a single node.
 
-We compose a network of two hidden layers to start off with something. We go by a scheme with 100 neurons in the first hidden layer and 50 neurons in the second layer. As activation function we settle on the `relu` function as a it proved very robust and widely used. To make our live easier later, we wrap the definition of the network in a method called `create_nn`.
+We compose a network of two hidden layers to start off with something. We go by a scheme with 100 neurons in the first hidden layer and 50 neurons in the second layer. As activation function we settle on the `relu` function as a it is very robust and widely used. To make our live easier later, we wrap the definition of the network in a function called `create_nn()`.
 
 ```python
 from tensorflow import keras
 
-def create_nn():
+def create_nn(X_data):
```

Comment:
Better!

Reply:
If you feel like it, you could consider having `input_shape` as a parameter instead of `X_data`. But we can live with this, it is better than what it was!